packages/core/src/external/carequality/ihe-gateway-v2/saml/security/security-header.ts

```diff
@@ -102,8 +115,11 @@ export function createSecurityHeader({
           "saml2:Attribute": [
             {
               "@_Name": "urn:oasis:names:tc:xspa:1.0:subject:subject-id",
-              "@_NameFormat": "urn:oasis:names:tc:SAML:2.0:attrname-format:uri",
```

Comment:
double checking that we don't need the format here anymore?

Reply:
not needed.

Reply:
🤦